react/features/filmstrip/actionTypes.ts

```diff
@@ -18,6 +18,15 @@ export const SET_FILMSTRIP_ENABLED = 'SET_FILMSTRIP_ENABLED';
  */
 export const SET_FILMSTRIP_VISIBLE = 'SET_FILMSTRIP_VISIBLE';
 
+/**
+ * The type of (redux) action which enlarges the filmstrip.
+ *
+ * {
+ *     type: RESIZE_FILMSTRIP,
+ * }
+ */
+export const RESIZE_FILMSTRIP = 'RESIZE_FILMSTRIP';
```

Comment:
Style: alpha sort.

Reply:
This rule is not really applied always, eg, I checked this file and we don't have the alpha sort.   Will it make sense to have this in eslint? I think https://eslint.org/docs/latest/rules/sort-vars

Reply:
For these files and constants file sure.